list-doi-requests/src/test/java/no/unit/nva/doirequest/list/ListDoiRequestsHandlerTest.java

```diff
@@ -181,47 +195,130 @@ public void listDoiRequestsReturnsEmptyListForUserWhenUserHasNoValidRole()
         assertThat(responseBody, is(emptyCollectionOf(Publication.class)));
     }
 
+    @Test
+    public void listDoiRequestsForUserReturnsDtosWithMessagesIncludedWhenThereAreDoiRequestMessagesForTheDoiRequest()
+        throws ApiGatewayException, IOException {
+        List<Publication> publications = createPublishedPublicationsOfSameOwner();
+        createDoiRequests(publications);
+        final var doiRequestMessages = creteDoiRequestMessagesForPublications(publications);
+
+        URI commonPublisherId = publications.get(FIRST_ELEMENT).getPublisher().getId();
+        String commonOwner = publications.get(FIRST_ELEMENT).getOwner();
+        List<String> actualMessages = sendRequestAndReadMessages(commonOwner, commonPublisherId, CREATOR_ROLE);
+
+        String[] expectedMessages = extractExpectedMessageTexts(doiRequestMessages);
+        assertThat(actualMessages, containsInAnyOrder(expectedMessages));
+    }
+
+    @Test
+    public void listDoiRequestsForCuratorReturnsDtosWithMessagesIncludedWhenThereAreDoiRequestMessagesForTheDoiRequest()
+        throws ApiGatewayException, IOException {
+        var publications = createPublishedPublicationsOfSamePublisherButDifferentOwner();
+        createDoiRequests(publications);
+        final var doiRequestMessages = creteDoiRequestMessagesForPublications(publications);
+
+        URI commonPublisherId = publications.get(0).getPublisher().getId();
+        List<String> actualMessages = sendRequestAndReadMessages(SOME_CURATOR, commonPublisherId, CURATOR_ROLE);
+
+        String[] expectedMessages = extractExpectedMessageTexts(doiRequestMessages);
+        assertThat(actualMessages, containsInAnyOrder(expectedMessages));
+    }
+
+    public List<String> sendRequestAndReadMessages(String userIdentifier, URI commonPublisherId, String curatorRole)
```

Comment:
should a helper method in a test class be public when its not annotated with Test?